common/app/routes/challenges/utils.js

```diff
@@ -46,15 +46,35 @@ export function createTests({ tests = [] }) {
     });
 }
 
+function logReplacer(value) {
+  if (typeof value === 'string' && !value.startsWith('//')) {
+    return '"' + value + '"';
+  } else if (typeof value === 'number' && isNaN(value)) {
```

Comment:
If you uses a `return` in the body of each `if` then you just omit the `else if` and just use a serious of `if`s

```js
if (...) {
  return ...
}
if (...) {
  return ...
}
return ...
```

this reduces the number of chars and gives a cleaner look.